### PR TITLE
Add Support for the 'hash' Output Format

### LINKED ISSUE #221: Add ability to use FFmpeg Hash
Hi there, 

Do you plan on adding ffmpeg's hashing ability?

https://ffmpeg.org/ffmpeg-formats.html#hash-1

If not could I submit a pr for this?

### PR DESCRIPTION
**Pull request by 06needhamt**
Closes #221 

This PR adds support for the 'hash' output format.

### CODE REVIEW

File: Xabe.FFmpeg.Test/ConversionTests.cs [hunk 1]
```diff
@@ -120,6 +120,28 @@ public async Task SetOutputPixelFormatTest()
             Assert.Equal("yuv420p", resultFile.VideoStreams.First().PixelFormat);
         }
 
+        [Fact]
```
Comment (tomaszzmuda) [inline Xabe.FFmpeg.Test/ConversionTests.cs]: Please change it to theory and test every value from HashFormat

Comment (06needhamt) [reply Xabe.FFmpeg.Test/ConversionTests.cs]: Done, I don't know if there is a way to improve this so we don't need the switch statement?

Comment (tomaszzmuda) [reply Xabe.FFmpeg.Test/ConversionTests.cs]: Just pass length as a second parameter to function :)
[InlineData("MD5", 37L)]
public async Task SetHashFormatTest(string hashFormat, long expectedLength)

File: Xabe.FFmpeg.Test/ConversionTests.cs [hunk 2]
```diff
@@ -120,6 +120,102 @@ public async Task SetOutputPixelFormatTest()
...
+        [InlineData("SHA160")]
+        [InlineData("SHA224")]
+        [InlineData("SHA256")]
+        [InlineData("SHA512/224")]
+        [InlineData("SHA512/256")]
+        [InlineData("SHA384")]
+        [InlineData("SHA512")]
+        [InlineData("CRC32")]
+        [InlineData("Adler32")]
+        public async Task SetHashFormatTest(string hashFormat)
```
Comment (tomaszzmuda) [inline Xabe.FFmpeg.Test/ConversionTests.cs]: Pass here second parameter

File: Xabe.FFmpeg.Test/ConversionTests.cs (hunk 1 above)
Comment (06needhamt) [reply Xabe.FFmpeg.Test/ConversionTests.cs]: Done

File: Xabe.FFmpeg.Test/ConversionTests.cs (hunk 2 above)
Comment (06needhamt) [reply Xabe.FFmpeg.Test/ConversionTests.cs]: Done

**Pull request merged**